Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Ride.hs

```diff
@@ -50,6 +52,8 @@ import qualified Storage.Queries.BookingCancellationReason as QBCReason
 import qualified Storage.Queries.CallStatus as QCallStatus
 import qualified Storage.Queries.DriverLocation as QDrLoc
 import qualified Storage.Queries.DriverQuote as DQ
+-- import qualified Storage.Queries.FareParameters as QFareParams
```

Comment:
Remove commented code. Please do it by yourself instead of waiting for someone to comment.